### ISSUE TITLE
The detection training loss shows significant fluctuations every other epoch

### ISSUE THREAD

**Issue by babywakeup** [Topdu/OpenOCR#198]
<img width="1169" height="495" alt="Image" src="https://github.com/user-attachments/assets/b4ed01ed-cf4c-4a77-a8ba-567d06d0ad11" />

It is training for detection. And  I added more icdar datasets besides your finetuning dataset. And The training loss shows significant fluctuations every other epoch. Is it normal?

**Comment by Topdu** [comment]
Please compare the loss without using more icdar datasets.
